src/commands/init.js

```diff
@@ -202,6 +213,71 @@ const addToPackageScripts = async (cwd: string) => {
   );
 };
 
+const addToGradleBuild = async (cwd: string) => {
```

Comment:
Why arrow function instead of regular function? We don't gain any benefit from arrow function here and in the whole codebase we use arrow function only when it makes sense or it's needed.

Reply:
I just copied the existing addToXcodeBuild arrow function as an exemplar.